lib/replace_parameters.js

```diff
@@ -583,7 +576,17 @@ module.exports = class ParameterReplacer {
                     };
                 } catch(e) {
                     console.error(example);
-                    throw e;
+                    return {
+                        id: example.id,
+                        type: example.type,
+                        flags: example.flags || {},
+                        context: example.context,
+                        utterance: example.utterance,
+                        preprocessed: example.preprocessed,
+                        target_code: example.target_code
+                    };
+                    // console.error(e);
+                    // throw e;
```

Comment:
No. If there is an error, you're expected to crash. Otherwise the error goes unnoticed and we end up with broken datasets. The try-catch is only to add extra debugging info.